WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -371,26 +360,42 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        initViewModel()
+        initViewModel(savedInstanceState)
 
         showPost()
     }
 
-    private fun initViewModel() {
+    private fun initViewModel(savedInstanceState: Bundle?) {
         viewModel = ViewModelProvider(this, viewModelFactory).get(ReaderPostDetailViewModel::class.java)
 
-        viewModel.uiState.observe(viewLifecycleOwner, { renderUiState(it) })
+        viewModel.uiState.observe(viewLifecycleOwner, {
+            uiHelpers.updateVisibility(text_error, it.errorVisible)
+            uiHelpers.updateVisibility(progress_loading, it.loadingVisible)
+            when (it) {
+                is LoadingUiState -> Unit // Do Nothing
+
```

Comment:
Minor (🔍): Consider removing the extra empty lines.

Reply:
Removed in 4b0bd83